apps/dsp-app/src/app/workspace/resource/representation/still-image/still-image.component.ts

```diff
@@ -227,19 +233,45 @@ export class StillImageComponent
             this._viewer.destroy();
             this._viewer = undefined;
         }
+        if (this.imagesSub) {
+            this.imagesSub.unsubscribe();
+        }
+
+        if (this.fileInfoSub) {
+            this.fileInfoSub.unsubscribe();
+        }
     }
 
     private loadImages() {
-        this._rs
+        // closing, so no more loading of short in between images if turning
+        // multiple pages
+        this._viewer.close()
+        if (this.imagesSub) {
+            this.imagesSub.unsubscribe();
```

Comment:
We really need state management for this component in the future. This is a hack at best, even though I understand its necessity.

Reply:
Indeed, the whole component needs refactoring. The hack follows the "busieness logic" ...